scripts/lib/CIME/XML/env_mach_specific.py

```diff
@@ -246,7 +246,6 @@ def _get_module_commands(self, modules_to_load, shell):
         cmds = []
         last_action = None
         last_cmd    = None
-        actions_that_cannot_be_batched = ["swap"]
 
```

Comment:
seems like a fix is being undone here?

Reply:
looks that way, but this is actually expected. Look below the comment and you'll see this variable is being set.

Reply:
i see, thanks